queue/src/fiskaltrust.Middleware.Localization.QueueES/Processors/InvoiceCommandProcessorES.cs

```diff
@@ -86,8 +86,16 @@ public async Task<ProcessCommandResponse> FullInvoiceRequestAsync(ProcessCommand
         {
             LastReceipt = lastReceipt,
         };
-        
+
         // Generate series identifier if not set
+        // Is there a convention for the fkt prefix?
+        // Why 1000?
+
+        // We'll have to check the supported length of the numseriesfactura in verifactu.
+
+        // That should not be initialized here but during the first initialization of the queue. Maybe in the bootstrapper or maybe in the start receipt.
+
+        // What is the cashboxidentification of the queue for if we don't use it here?
```

Comment:
The CashBoxIdentification is the identifier of the so called device. The series is not necessarily the device which is why those things are seperated